store/auth.js

```diff
@@ -41,6 +41,11 @@ export const actions = {
   register({ commit, dispatch }, { email, username, password, role }) {
     return this.$services.users.create({ email, username, password, role })
   },
+  update({ commit }, { id, username, password }) {
+    return this.$services.users
+      .update({ username, password }, id)
+      .then(res => {})
```

Comment:
E se for o usuário logado, vai atualizar os dados dele?

Reply:
Vai sim, ele passa o Id na req.